src/main/java/seedu/cakecollate/logic/parser/AddCommandParser.java

```diff
@@ -52,8 +53,9 @@ public AddCommand parse(String args) throws ParseException {
                 ParserUtil.parseOrderDescriptions(argMultimap.getAllValues(PREFIX_ORDER_DESCRIPTION));
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         DeliveryDate deliveryDate = ParserUtil.parseDeliveryDate(argMultimap.getValue(PREFIX_DATE).get());
+        Request request = new Request("Give me more pineapples.");
 
-        Order order = new Order(name, phone, email, address, orderDescriptionSet, tagList, deliveryDate);
+        Order order = new Order(name, phone, email, address, orderDescriptionSet, tagList, deliveryDate, request);
```

Comment:
Wait so whenever the input is an add command the request is initialised to "Give me more pineapples."?

Because you implemented request seperately and not in the add command, do you need the request in the addCommand parse function?

Reply:
it should be initialized as an empty string, thanks for catching that, i forgot to test the addcommand